vocode/streaming/synthesizer/google_synthesizer.py

```diff
@@ -75,7 +75,7 @@ async def create_speech(
         in_memory_wav.setnchannels(1)
         in_memory_wav.setsampwidth(2)
         in_memory_wav.setframerate(output_sample_rate)
-        in_memory_wav.writeframes(response.audio_content)
+        in_memory_wav.writeframes(response.audio_content[44:])
```

Comment:
What's the significance of 44? I'm guessing there's some start sequence that's always prepended to the audio content? Is the audio content guaranteed to have a length of at least 44?

Reply:
44 is the size of the wave header, it shouldn't be a magic number really, but is okay for now